syntax = "proto3";

package hello;

service HelloService {
    rpc SayHello(HelloRequest) returns (HelloResponse);
    rpc SayHelloStream(HelloRequest) returns (stream HelloResponse);
}

message HelloRequest {
    string name = 1;
    int32 age = 2;
}

message HelloResponse {
    string message = 1;
    int64 timestamp = 2;
}
